server/routes/articleRoutes.js

```diff
@@ -6,6 +6,7 @@ import authMiddleware from '../middlewares/authMiddleware';
 const router = express.Router();
 
 router.get('/feeds', articleController.getAllArticles);
+router.get('/myarticles', authMiddleware, articleController.getMyArticles);
```

Comment:
how  do you know the user whos articles you want to fetch ??

Reply:
Yes because I passed the jwt decoded data of the user in a request from the `server/middlewares/authMiddleware.js` which is the middleware for this route

Reply:
okay noted